ipywidgets_bokeh/src/webpack.ts

```diff
@@ -10,3 +10,5 @@ if (script instanceof HTMLScriptElement) {
     __webpack_require__.p = url
   }
 }
+
+export {}
```

Comment:
I'm assuming this is required because of the new `isolatedModules` flag in `tsconfig.json`.